lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={props.tab === props.selectedTab ? 'tab active-tab' : 'tab'}
       onClick={() => {
```

Comment:
well done here with the ternary, you could also have done it with a template string like this {`tab ${props.tab === props.selectedTab ? "active-tab" : "" `}